mce-label-injector/deployments/mutatingwebhook.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: mce-label-injector-service
+  labels:
+    app: mce-label-injector
+spec:
+  ports:
+  - port: 9443
+    targetPort: 9443
+  selector:
+    app: mce-label-injector
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: mce-label-injector-configuration
+  annotations:
+    cert-manager.io/inject-ca-from: default/mce-label-injector
+webhooks:
+- admissionReviewVersions:
+  - v1
+  - v1beta1
+  clientConfig:
+    caBundle: Cg==
+    service:
+      name: mce-label-injector-service
+      namespace: "default"
+      path: /mutate
+      port: 9443
+  failurePolicy: Fail
+  sideEffects: None
+  name: mce-label-injector.x-k8s.io
+  rules:
+  - apiGroups:
+    - cluster.x-k8s.io
```

Comment:
There is also, `ipam.cluster.x-k8s.io` , `runtime.cluster.x-k8s.io` and `addons.cluster.x-k8s.io` , better to check the apiVersion as well